Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/pdf/ui/exportAll/DccExportAllOverviewFragment.kt

```diff
@@ -92,17 +97,21 @@ class DccExportAllOverviewFragment : Fragment(R.layout.fragment_dcc_export_all_o
             .setMessage(R.string.export_all_error_message)
             .setNeutralButton(R.string.export_all_error_faq) { _, _ ->
                 openUrl(R.string.certificate_export_all_error_dialog_faq_link)
-            }.setPositiveButton(android.R.string.ok) { _, _ -> }
-            .setOnDismissListener { popBackStack() }
+            }.setPositiveButton(android.R.string.ok) { _, _ -> popBackStack() }
```

Comment:
Is this necessary?

Reply:
No, thanks for pointing out. Removed this call.